src/components/ToggleGroup.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { ToggleButton, ToggleButtonGroup } from '@mui/material'
+import { useGuesses } from '../context/GuessContext' // see line 11
+
+
+export const ToggleGroup = ({ values, labels, handleChange }) => {
+
+    const { name, value1, value2, value3 } = values
+    const { groupLabel, buttonLabel1, buttonLabel2, buttonLabel3 } = labels
+
+    const { guesses } = useGuesses() // ? pass as prop from parent [also has useGuesses()] instead?
```

Comment:
In this case its better to useContext as we're already passing quite a few props. Plus the whole point of our context here is to minimise passing the same props all the time